modules/apps/frontend-editor/frontend-editor-ckeditor-web/src/main/resources/META-INF/resources/editor/BalloonEditor.js

```diff
@@ -20,12 +20,23 @@ import DEFAULT_BALLOON_EDITOR_CONFIG from './config/DefaultBalloonEditorConfigur
 
 import '../css/main.scss';
 
-const BalloonEditor = ({config = {}, contents, name, ...otherProps}) => {
+const EMPTY_OBJECT = {};
+
+const BalloonEditor = ({
+	config = EMPTY_OBJECT,
+	contents,
+	name,
+	...otherProps
+}) => {
 	const editorConfig = {
 		...DEFAULT_BALLOON_EDITOR_CONFIG,
 		...config,
 	};
 
+	if (!editorConfig.enableBalloonEditor) {
+		return null;
+	}
```

Comment:
I was initially thinking we should throw an error in this case, but this is fine as well :+1: